docker-compose.yml

```diff
@@ -48,6 +50,14 @@ services:
   # Shared Services #
   ###################
 
+  redis:
+    image: redis:latest
+    restart: unless-stopped
+    ports:
+    - 6379:6379
+    volumes:
+      - redis_data_vol:/data
```

Comment:
A few notes here:
* In case we have other Redis in the future and we decide(?) to use a dedicated Redis for Session, maybe naming this `django-session-redis` or something similar?
* Pinging a version may be better to avoid incompatibility when `redis:latest` is updated.
* I haven't looked into it but can you check if we want to add `stdin_open: true`?

Reply:
- We could use the same redis instance for all local usage. `django-redis` prepends prefix to the key so we don't have to worry about collisions with keys for other local usage.
- Yeah, pining a version would be ideal. I'll have to ask what version we're planning to use for prod/staging.
- Not if we don't need interactive shells I think.

Reply:
* Ah, make sense. In addition, `docker-compose` is only for local env only too and no need to run multiple.
* OK
* Yeah, let's skip it until we need it